libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/RustMatrixClient.kt

```diff
@@ -94,13 +82,15 @@ class RustMatrixClient constructor(
 
     override val sessionId: UserId = UserId(client.userId())
 
+    private val roomListService = client.roomListService()
```

Comment:
What it the diff between `roomListService` and `roomListServiceWithEncryption`?

Reply:
Actually, we should use `roomListServiceWithEncryption` as long as we don't have a dedicated sync for encryption